site/src/components/roadmap/style.module.css

```
@@ -0,0 +1,81 @@
+.heading {
+  margin-bottom: 5%;
+  text-align: left;
+  display: inline-flex;
+  align-items: center;
+  white-space: nowrap;
+}
+
+.heading4 {
+  margin: auto;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.cardContainer:global(.saltGridLayout) {
+  margin-top: 50px;
+  gap: 20px;
+}
+
+.content {
+  flex: 1;
+  flex-direction: column;
+}
+
+.searchInput {
+  flex: 1;
+  max-width: 400px;
+  width: 100%;
+}
+
+.date {
+  display: inline;
+}
+
+.column {
+  padding-right: 20px;
+}
+
+.cardFuture {
+  width: 400px;
+  margin-bottom: 15px;
+  min-height: unset;
+  border-left-color: var(--salt-palette-neutral-secondary-foreground);
+}
+
+.cardInProgress {
+  width: 400px;
+  margin-bottom: 15px;
+  min-height: unset;
+  border-left-color: var(--salt-palette-info-border);
```

```suggestion
  border-left-color: var(--salt-status-info-borderColor);
```
characteristics should be used directly, never palette